docs/content/preview/deploy/kubernetes/single-zone/oss/helm-chart.md

```diff
@@ -259,7 +259,7 @@ kubectl get services --namespace yb-demo
 Expect output similar to the following:
 
 ```output
-NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                                        AGE
+NAME                 TYPE           CLUSTER-IP      EXTERNAL-IP        PORT(S)                                        AGE
```

Comment:
Where did this one come from? And does it need to be replicated in all the other helm-chart files in this and the other versions?

Reply:
@polarweasel On couple of instances, it is already aligned. Let me check all the helm-chart files.

Reply:
@polarweasel Applied this change in couple of other instances (especially in helm-chart files) in all versions.